File: cpp/CW_Highest_and_Lowest.cpp
#include <bits/stdc++.h>
using namespace std;

std::string highAndLow(const std::string &numbers)
{
  std::istringstream iss(numbers);
  std::vector<int> nums = {istream_iterator<int>(iss), {}};
  std::sort(nums.begin(), nums.end());
  return  to_string(nums.back()) + " " + to_string(nums.front());
}

int main()
{
  auto res = highAndLow("8 3 -5 42 -1 0 0 -9 4 7 4 -4");
  std::cout << "res: " << res << '\n';
  return (0);
}

File: cs/CW_Highest_and_Lowest.cs
using System;
using System.Linq;

var res = Kata.HighAndLow("1 2 3 4 5");
Console.Error.WriteLine($"res: {res}"); 

public static class Kata
{
  public static string HighAndLow(string numbers)
  {
    var nums = numbers.Split().Select(int.Parse).Order();
    return nums.Last() + " " + nums.First();
  }
}
